src/app/components/tv-show/tv-show.component.ts

```diff
@@ -10,9 +11,12 @@ import { TvShow } from '../../consts/tv-show';
 export class TvShowComponent implements OnInit {
   show: TvShow = null;
 
-  constructor(private route: ActivatedRoute) { }
+  constructor(private route: ActivatedRoute, private sanitizer: DomSanitizer) { }
```

Comment:
Why do you need sanitizer here?

Reply:
Hm... I think I forgot to remove it from here after adding the pipe and using DomSanitizer there.